README.md

```diff
@@ -37,16 +37,16 @@ Versions match. As an example, to get Petals Cockpit 0.20.0, just type in `docke
 ## Run parameters
 
 When running Petals Cockpit by a docker image, a default administrator user is added automatically:
-* user: `admin`
 * username: `admin`
+* user: `admin`
 * password: `admin`
 
 It is possible to change this user with the following parameters: 
 
 | Argument | Optional | Default | Description |
 | -------- | :------: | :-----: | ----------- |
-| COCKPIT_USER | yes | `admin` | The user's id, also his login. |
-| COCKPIT_USERNAME | yes | `admin` | The name under which the user will appear. |
+| COCKPIT_USERNAME | yes | `admin` | The user's id, also his login. |
+| COCKPIT_NAME | yes | `admin` | The name under which the user will appear. |
```

Comment:
Or you could consider using **COCKPIT_USER_DISPLAY_NAME** for this one.

Reply:
`name` is the term used in cockpit for the user display name. But I guess `COCKPIT_USER_DISPLAY_NAME` is close enough and clearer.